src/support.js

```diff
@@ -518,13 +518,15 @@ function loadWebAssemblyModule(binary, flags) {
         };
       }
     };
+    var proxy = new Proxy(env, proxyHandler);
     var info = {
       global: {
         'NaN': NaN,
         'Infinity': Infinity,
       },
       'global.Math': Math,
-      env: new Proxy(env, proxyHandler),
+      env: proxy,
+      wasi_unstable: proxy,
```

Comment:
`env:` should be `'env':` for closure safety

Reply:
But its already a naked env and that seems to work ok.  How is that not a problem already?   Perhaps we can investigate and look into that as a separate change?

Reply:
Possibly it just doesn't hit an issue with closure, but it's at risk of doing that. Yes, we can look into this separately.